quiz.rb

```diff
@@ -1,20 +1,49 @@
 # Question 1
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
+class Dog
+  attr_reader :name, :breed, :age
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+end
 
 # Question 2
 # Create a new Dog object.
+tuffy = Dog.new("tuffy","pomeranian",3)
 
 # Question 3
 # Create a Person class, with attributes name, age, and location.
+
+
 # It should set all three attributes when you create a new Person.
 # Name and location should be writeable, but age should be read-only.
-
+class Person
+  attr_reader :age
+  attr_accessor :name, :age
```

Comment:
Whoops - I think you meant to write `:location` there, but instead you wrote `:age` twice.